### PR TITLE
Add fun images of candidates to bingo overlay

### PR DESCRIPTION
**Pull request by andrewjtimmons**
Solves #16 

We may want different images, but this gives layout that works and lets us know we can just pop in other square images later.

Used clinton for mixed board win as I wasn't sure if an image of the two candidates together was desirable.  

screen shot 2016-09-18 at 12 19 54 pm

screen shot 2016-09-18 at 12 21 42 pm

screen shot 2016-09-18 at 12 22 25 pm

### CODE REVIEW

File: src/BingoCard.js [hunk 1]
```diff
@@ -43,12 +43,17 @@ export default class BingoCard extends React.Component {
         });
     }
     render() {
+        let bingoImagePath = `images/${this.props.type}_win_sq_resized.png`;
+        console.log(bingoImagePath);
```
Comment (dglazkov) [inline src/BingoCard.js]: remove the logging.

File: src/BingoCard.js [hunk 2]
```diff
@@ -43,12 +43,17 @@ export default class BingoCard extends React.Component {
...
         let bingoOverlay = !this.state.isBingo ? null : (
             <div className="bingoOverlay">
                 <h2>Bingo!</h2>
                 <p>
                     <a href="#" onClick={this.props.onPlayAgainRequested}>Play again</a>
                 </p>
+                <div className="bingoImageOverlay">
+                    <img src={bingoImagePath} className="bingoImage" alt="image of Hillary Clinton" />
```
Comment (dglazkov) [inline src/BingoCard.js]: The alt image text will be wrong when Trump or Mixed type chosen, right?

File: src/styles.scss [hunk 3]
```diff
@@ -38,7 +38,7 @@ td {
     bottom: 0;
     left: 0;
     right: 0;
-    padding-top: 40%;
+    padding-top: 2 em;
```
Comment (dglazkov) [inline src/styles.scss:41]: Let's pick where we're using ems or px on this site and use them consistently. Currently, it's mostly px. I don't mind going with ems. @JonathanKeslin, do you have a preference?

**Review by dglazkov — COMMENTED** [review]
This looks great, except for the alt text comment.

File: src/BingoCard.js (hunk 2 above)
Comment (andrewjtimmons) [reply src/BingoCard.js]: Good point.  Updated to be generic

**Review by dglazkov — APPROVED** [review]
LGTM

File: src/styles.scss (hunk 3 above)
Comment (andrewjtimmons) [reply src/styles.scss:41]: I will wait to Merge until we hear from @JonathanKeslin

**Review by decompil3d — APPROVED** [review]
Looks great - thanks! My only question is about image copyright, but we can solve that in a later change. Go ahead and merge.

**Comment by andrewjtimmons** [comment]
Good point on the images, perhaps the campaign has images we can use?  Should we reach out to Remy?

**Pull request merged**